packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -312,6 +400,20 @@ class ReorderableListView extends StatefulWidget {
   ///  hovering, and [SystemMouseCursors.grabbing] when dragging.
   final MouseCursor? mouseCursor;
 
+  /// The builder for separator widgets when using the [ReorderableListView.separated] constructor.
+  ///
+  /// This callback is used to build the separator widgets that appear between items.
+  /// It is called with indices from 0 to `itemCount - 2` (where `itemCount` is
+  /// the number of actual data items).
+  ///
+  /// Will be null if not using the `separated` constructor.
```

Comment:
```suggestion
  /// Null if this is not a `ReorderableListView.separated`.
```

or something similar.